waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -607,13 +617,30 @@ proc relayConnectivityLoop*(pm: PeerManager) {.async.} =
     await pm.connectToRelayPeers()
     await sleepAsync(ConnectivityLoopInterval)
 
+proc logSummary*(pm: PeerManager) {.async.} =
+  heartbeat "Log peer manager summary", LogSummaryInterval:
+    let (inRelayPeers, outRelayPeers) = pm.connectedPeers(WakuRelayCodec)
+    let maxConnections = pm.switch.connManager.inSema.size
```

Comment:
I wonder if it may be also interesting to print the in/out ratio, for debugging purposes.

Reply:
its there right? or wdym? we have `inPeersTarget` and `outPeersTarget`

```
inRelayConns = $inRelayPeers.len & "/" & $inPeersTarget,
outRelayConns = $outRelayPeers.len & "/" & $pm.outPeersTarget,
```

Reply:
Morning! I mean to print the ratio that we expect. This is the real ratio that is being applied. I would print the theoretical ratio just in case it is not accomplished by the real one, for any reason.
Not a big deal though. If you consider we have enough info, then go ahead :)

Reply:
I see. if you dont mind i leave it as it is, since whats being printed here its more complete imho. eg: "...5/10, 30/50" which indicates that 5 `in` slots out of 10 are taken and 30 out of 50 `out` are taken.

Reply:
Ok cool, is perfect then! thanks !